src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,422 @@
+  into text.
+- Forms applications where users can use pen or finger input due to time constraints.
+- Games that require filling in letters or numbers, such as crosswords, hangman, or sudoku.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | [In progress][ot]        |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use the Handwriting Recognition API {: #use }
+
+### Enabling via about://flags
+
+To experiment with the Handwriting Recognition API locally, without an origin trial token, enable
+the `#experimental-web-platform-features` flag in `about://flags`.
+
+{% Aside 'caution' %} Note that the API is currently exclusive to Chrome&nbsp;OS devices.
+{% endAside %}
+
+### Feature detection
+
+Browser support can be detected by checking for the existence of the `createHandwritingRecognizer()`
+method on the navigator object:
+
+```js
+if ('createHandwritingRecognizer' in navigator) {
+  // 🎉 The Handwriting Recognition API is supported!
+}
+```
+
+### Core concepts
+
+<!-- 🎉TODO🎉: Image showing the core entities -->
+
+The Handwriting Recognition API converts handwritten input into text, regardless of the input method
+(mouse, touch, pen). The API has four main entities:
+
+1. A _point_ represents where the pointer was at a particular time.
+2. A _stroke_ consists of one or more points. The recording of a stroke starts when the user puts
+   the pointer down (i.e., clicks the primary mouse button, or touches the screen with their pen or
+   finger) and ends when they raise the pointer back up.
+3. A _drawing_ consists of one or more strokes. The actual recognition takes place at this level.
+4. The _recognizer_ is configured with the expected input language. It is used to create an instance
+   of a drawing with the recognizer configuration applied.
+
+#### Creating a recognizer
+
+To recognize text from handwritten input, you need to obtain an instance of a
+`HandwritingRecognizer` by calling the `createHandwritingRecognizer()` method on the `navigator`
+object, and passing constraints to it. They are used to determine the handwriting recognition model
+that should be used. Currently, you can specify a list of languages in order of preference. In the
+following example, the developer would prefer to recognize texts in English and fall back to German
+as spoken in Germany when available.
+
+```js
+const recognizer = await navigator.createHandwritingRecognizer({
+  languages: ['en', 'de-DE'],
```

Comment:
I think we should omit 'de-DE'.

At the moment, Chrome OS recognizers only support one language (providing multiple languages will cause an error); and the supported language is 'en' (English)

Reply:
The text describes how to deal with multiple languages. The text would need to be rewritten as well. 

How soon will this be enabled? It would be better to add a disclaimer that this isn't enabled yet, then to remove the section and add it back later. Removing the disclaimer will take all of about five minutes. Re-adding text would take a whole review process.

Reply:
@yuhonglin better knows about the timeline. But probably not in near future.

Reply:
Yeah, multi-language support is non-trivial and may need one-quarter to implement. I will try make it as an OKR for next quarter.

Reply:
I removed the 'de-DE' sample and added a note about the gesture support in the aside box.